### PR TITLE
need to review my homework

### PR DESCRIPTION
**Pull request by Rohullah-Mohammadi**
1.  What was most challenging about the homework so far? 
  * Answer: the most challenging part was to remember and understand which command need to use in 
    terminal.
___
  2.  How did you solve it?
  * answer:  by asking questions from dear teacher @nikosolihin , googling and watching videos on 
     YouTube.

### CODE REVIEW

File: terminal.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+**Try it: cd and cd ..**
+
+1. To go back from work directory to "to_do" type (cd ..)
+
+2. To go back from projects_to_delegate directory to "to_do" type cd .. and once again cd .. or we can combine them and type cd ../.. just one time.
```
Comment (nikosolihin) [inline terminal.md:5]: Correct – `cd ../..` works. Even if you omit the `cd`, your terminal will understand what `..` is. Just a tip.

File: terminal.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+3. To go from home directory to projects_to_delegate type cd .. then (cd work) then (cd projects_to_delegate) or just combine and type (cd ../work/projects_to_delegate)
+
+4. To go back from projects_to_delegate to home directory type cd .. again cd .. and (cd home) or just combine and type (cd ../../home)
```
Comment (nikosolihin) [inline terminal.md:9]: Nice move!

File: terminal.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+**Try it: Removing files (rm) and directory (rm -rf)**
+1. random.txt is a file inside to_do directory to delete a file use (rm) now type (rm random.txt)
+
+2. to remove a directory need to type (rm -rf) so home is a directory, now type (rm -rf home)
+
+3. to coming back from work directory to remove home directory type cd .. and then type (rm -rf home )
+
+4. to remove projects_to_delegate directory type cd .. and then type (rm -rf projects_to_delegate)
+
+5. to remove cleaning.txt file in home directory type cd .. again cd.. and (cd home) and (rm cleaning.txt) or just simple type cd ../../home and (rm cleaning.txt)
```
Comment (nikosolihin) [inline terminal.md:20]: Nice. I think you clearly get the differences between `rm -rf` and just `rm`.

**Review by nikosolihin — CHANGES_REQUESTED** [review]
Nice work @Rohullah-Mohammadi.

Just missing the reflection questions then we can merge the answer to master.

File: terminal.md (hunk 2 above)
Comment (Rohullah-Mohammadi) [reply terminal.md:9]: Thank you

**Pull request merged**